pvfactors/geometry/base.py

```diff
@@ -122,7 +122,7 @@ def _get_rotation_from_tilt_azimuth(surface_azimuth, axis_azimuth, tilt):
     # Calculate rotation of PV row (signed tilt angle)
     is_pointing_right = ((surface_azimuth - axis_azimuth) % 360.) > 180.
     rotation = np.where(is_pointing_right, tilt, -tilt)
-
+    rotation = np.where(tilt == 0, -0.0, rotation)  # GH 125
```

Comment:
wow I have no idea what's going on here 😅 but your test definitely shows that it's working so great catch! 🚀

Reply:
@kanderso-nrel I don't know what array sizes we are dealing with here, but this is likely faster and more memory efficient:
```python
rotation[tilt==0] = -0.0
```